picard/ui/options/renaming.py

```diff
@@ -126,6 +127,23 @@ def __init__(self, parent=None):
         disabled_color = self.script_palette_normal.color(QPalette.Inactive, QPalette.Window)
         self.script_palette_readonly.setColor(QPalette.Disabled, QPalette.Base, disabled_color)
         self.ui.scripting_documentation_button.clicked.connect(self.show_scripting_documentation)
+        self.ui.example_filename_sample_files_button.clicked.connect(self._sample_example_files)
+        self._sampled_example_files = []
+
+        # Sync example lists vertical scrolling
+        def _sync_toolbar_vert(value):
+            self.ui.example_filename_before.blockSignals(True)
+            self.ui.example_filename_after.blockSignals(True)
+            self.ui.example_filename_before.verticalScrollBar().setValue(value)
+            self.ui.example_filename_after.verticalScrollBar().setValue(value)
+            self.ui.example_filename_before.blockSignals(False)
+            self.ui.example_filename_after.blockSignals(False)
+
+        self.ui.example_filename_before.verticalScrollBar().valueChanged.connect(
+            _sync_toolbar_vert)
+
+        self.ui.example_filename_after.verticalScrollBar().valueChanged.connect(
+            _sync_toolbar_vert)
```

Comment:
I think `_sync_toolbar_vert` was meant to be `_sync_scroolbar_vert`.
Also there is no need to block signals and set value of the scrollbar emiting the signal.


We could make it more generic and working for more than 2 scrollbars:

```python
        def sync_vertical_scrollbars(widgets):
            """Sync position of vertical scrollbars for listed widgets"""
            def _sync_scrollbar_vert(widget, value):
                widget.blockSignals(True)
                widget.verticalScrollBar().setValue(value)
                widget.blockSignals(False)

            widgets = set(widgets)
            for widget in widgets:
                for other in widgets - {widget}:
                    widget.verticalScrollBar().valueChanged.connect(
                        partial(_sync_scrollbar_vert, other))

        # Sync example lists vertical scrolling
        sync_vertical_scrollbars((self.ui.example_filename_before, self.ui.example_filename_after))
```

`sync_vertical_scrollbars()` could be then moved to `picard/ui/util.py`